apps/flo-cli/src/adapters/git/git.utils.ts

```
@@ -59,3 +57,14 @@ export const getWorktreeFromBranch = (
 
     return worktree
 }
+
+export const assertGitHubInstalled = () => {
+    const sysCallService = SysCallService.getInstance()
+    if (sysCallService.testCommand('gh --version')) return
+
+    Logger.getInstance().error(
+        'Please install gh cli with `brew install gh` or go here: https://cli.github.com/manual/installation'
+            .red,
+    )
+    process.exit(1)
```

We should just throw an error here